aircraft/performance_tutorial.py

```diff
@@ -0,0 +1,104 @@
+""" A simple aircraft performance model to be used as GPkit tutorial"""
+from gpkit import Variable, Model
+from gpkit.tools import te_exp_minus1
+import numpy as np
+
+# Free Variables
+b      = Variable('b', 'm', 'Wing span')
+CD     = Variable('C_D', '-', 'Drag coefficient')
+CL     = Variable('C_L', '-', 'Lift coefficient')
+LDl    = Variable('\\frac{L}{D}_land', '-', 'Lift-to-drag in final descent')
+M      = Variable('M', '-', 'Mach number')
+Mdd    = Variable('M_{dd}', '-', 'Drag divergence Mach number')
+R      = Variable('R', 'nautical_miles', 'Range')
+S      = Variable('S', 'm^2', 'Wing reference area')
+s_GR   = Variable('s_{GR}', 'm', 'Ground roll distance')
+s_TO   = Variable('s_{TO}', 'm', 'Total takeoff distance')
+sl     = Variable('sl', 'm', 'Total landing distance')
+sl_gr  = Variable('sl_{gr}', 'm', 'Ground roll distance (landing)')
```

Comment:
variable names: the difference between `s_GR` and `sl_gr` is a bit confusing / subtle